frontend/src/lib/api/config.ts

```diff
@@ -0,0 +1,41 @@
+import { request } from "./request";
+
+export type GetArchiveConfigReturn = {
```

Comment:
Why the method signature in the type? The type is just a configuration right? regardless of it's origin?

I think we should maintain the same name as on the backend. So iether `Configuration` or `ArchiveConfiguration`, with of course methods respecting that name.